packages/prisma/schema.prisma

```diff
@@ -341,6 +341,7 @@ model Team {
   brandColor           String?
   darkBrandColor       String?
   verifiedNumbers      VerifiedNumber[]
+  bannerUrl            String?
```

Comment:
We need to modify the Avatar model to accept a new relationship for banners specifically asl well

Reply:
why exactly? we also use Avatar model to store team logo/ org logo without any relation 

/api/avatar/[uuid] -> gets image based on objectKey.

Reply:
cc: @zomars